site/content/en/docs/enterprise/social-accounts-configuration.md

```diff
@@ -106,32 +106,72 @@ There are 2 basic steps to enable GitHub account authentication.
 > but don't forget to add required permissions.
 > <br>In the **Permission** > **Account permissions** > **Email addresses** must be set to **read-only**.
 
-## Enable authentication with an Amazon Cognito
-
-To enable authentication, do the following:
-
-1. Create a user pool. For more information,
-   see [Amazon Cognito user pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
-2. Fill in the name field, set the homepage URL (for example: `https://localhost:8080`),
-   and authentication callback URL (for example: `https://localhost:8080/api/auth/social/amazon-cognito/login/callback/`).
-3. Create configuration file in CVAT:
-
-   1. Create the `auth_config.yml` file with the following content:
-
-   ```yaml
-   ---
-   social_account:
-     enabled: true
-     amazon_cognito:
-       client_id: <some_client_id>
-       client_secret: <some_client_secret>
-       domain: https://<domain-prefix>.auth.us-east-1.amazoncognito.com
-   ```
-
-   2. Set `AUTH_CONFIG_PATH="<path_to_auth_config>` environment variable.
-
-3. In a terminal, run the following command:
-
-   ```bash
-   docker compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.override.yml up -d --build
-   ```
+## Enable authentication with Amazon Cognito
+
+To enable authentication with Amazon Cognito for your CVAT instance, follow these steps:
+
+1. Create an **[Amazon Cognito pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)**
+   (_Optional_)
+1. Set up a new app client
+1. Configure social authentication in CVAT
+
+Now, let’s dive deeper into how to accomplish these steps.
+
+### Amazon Cognito pool creation
+
+This step is optional and should only be performed if a `user pool` has not already been created.
```

Comment:
Why use code notation for "user pool"? It's not an identifier or a code fragment, it's just a term.